source/pages/Home/Home.tsx

```diff
@@ -26,6 +25,10 @@ export const Home = () => {
     (state: RootState) => state.vault.activeAccount
   );
 
+  const currentFiatCurrency = useSelector(
+    (state: RootState) => state.price.fiat.asset
```

Comment:
we need to get the coins from coingecko api for web3 networks, syscoin core tickers are limited so this will not work for web3 networks